components/autosize_textarea.tsx

```diff
@@ -17,42 +16,53 @@ type Props = {
 
 export default class AutosizeTextarea extends React.PureComponent<Props> {
     private height: number;
+    private textareaRef: React.RefObject<HTMLTextAreaElement>;
```

Comment:
minor nit
```suggestion
    private textAreaRef: React.RefObject<HTMLTextAreaElement>;
```
we try to do camelCase throughout the code, so in order to be consintent we should rename it.